mk/containerized.mk

```diff
@@ -23,6 +22,8 @@ build: DOCKER_COMPOSE_ENV = DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1
 push: ## Push SVC container images to a registry. Requires previous "docker login"
 
 up: ## Launch services
+up: SVC ?= ceph nvmeof ## Services
+up: OPTS ?= --abort-on-container-exit --exit-code-from $(SVC) --remove-orphans --scale nvmeof=$(SCALE)
```

Comment:
I originally removed this from this file, as I wanted `containerized.mk` to be deployment-neutral and potentially reusable (there are no specific references to nvmeof stuff, just docker and docker-compose).

Reply:
Thank you for explaining, moving `up` back to Makefile.